paddle/fluid/framework/ir/auto_mixed_precision_pass.cc

```diff
@@ -669,7 +669,9 @@ bool AutoMixedPrecisionPass::InputVarsNotConvert(
     if (std::find(vecs.begin(), vecs.end(), var_name) != vecs.end()) {
       return true;
     }
-  } else if (GetOpOriginalType(op_desc->Type()) == "instance_norm") {
+  } else if (GetOpOriginalType(op_desc->Type()) == "instance_norm" ||
+             GetOpOriginalType(op_desc->Type()) == "layer_norm" ||
```

Comment:
no ln & gn
otherwise gn:
InvalidArgumentError: The type of data we are trying to retrieve (float32) does not match the type of data (float16) currently contained in the container.
  [Hint: Expected dtype() == phi::CppTypeToDataType<T>::Type(), but received dtype():15 != phi::CppTypeToDataType<T>::Type():10.] (at /home/model/dev/Paddle/paddle/phi/core/dense_tensor.cc:159)
  [operator < pd_kernel.phi_kernel > error]